WooCommerce/Classes/ViewRelated/Dashboard/Blaze/BlazeCampaignDashboardView.swift

```diff
@@ -205,6 +230,7 @@ private extension BlazeCampaignDashboardView {
 
 private extension BlazeCampaignDashboardView {
     enum Layout {
+        static let margin: CGFloat = 16
```

Comment:
I think `margin` is not used anywhere and can be removed.

Reply:
Removed in a9259c8d3d942e24dc76450cc3d53cae980e8cda.